test/unit/vagrant/ui_test.rb

```diff
@@ -231,16 +231,8 @@
 
   [:detail, :warn, :error, :info, :output, :success].each do |method|
     describe "##{method}" do
-      it "outputs UI type to the machine-readable output" do
-        expect(subject).to receive(:safe_puts).with(any_args) { |message|
-          parts = message.split(",")
-          expect(parts.length).to eq(5)
-          expect(parts[1]).to eq("")
-          expect(parts[2]).to eq("ui")
-          expect(parts[3]).to eq(method.to_s)
-          expect(parts[4]).to eq("foo")
-          true
-        }
+      it "does not output UI type to the machine-readable output" do
+        expect(subject).to receive(:safe_puts).never
```

Comment:
We would expect the UI output to still be included within the machine readable output. A new test should also be added which validates the new machine readable content is output as expected.